src/ontology/cl-edit.owl

```diff
@@ -27379,6 +27403,26 @@ AnnotationAssertion(oboInOwl:inSubset obo:CL_4023011 <http://purl.obolibrary.org
 AnnotationAssertion(rdfs:label obo:CL_4023011 "Lamp5 GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4023011 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 <https://identifiers.org/ensembl:ENSMUSG00000027270>)))
 
+# Class: obo:CL_4023012 (near-projecting glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023012 "A glutamatergic neuron located in the cerebral cortex that projects axons locally rather than long distance")
```

Comment:
Add full-stop. 

Not sure that "long-distance" is the correct contrast with "locally" -- maybe "distantly" works better simply for readability.